README.md

```diff
@@ -0,0 +1,30 @@
+# Project Name: HTML CSS Project
+
+Description: It's an html and css project with linters
+
```

Comment:
- [ ] Please your Pull Request (PR) description should always be detailed, and easy to understand. I suggest that you kindly update this PR description and re-submit for approval. Also, always ensure that your project description in README files is very descriptive enough. 😄

Reply:
Please, I have changed the description in my last commit, I will be grateful if you can help me check my last commit because the project is just to have the knowledge about setting up linters in our projects and be familiar with them. Thanks